scripts/inventory.gd

```diff
@@ -0,0 +1,44 @@
+extends Control
+onready var itemlist:ItemList = $CanvasLayer/ItemList
+var items:Dictionary = {
+	"Knife": {
+	'picture': 'res://assets/items/knife.png',
+	"description": 'A standard knife',
+	},
+	"Hell Knife": {
+	'picture': 'res://assets/items/hell_knife.png',
+	"description": 'The most powerful knife',
+	},
+	"Wooden Knife": {
+	"description": 'A knife made out of wood, not very effective...',
+	'picture': 'res://assets/items/wooden_knife.png',
+	}
+}
+func _ready():
+	$CanvasLayer.visible = false #<---- I can't do this in editor, so I have to do it via code.
+	additem('Knife')
+	additem('Wooden Knife')
+	additem('Hell Knife')
+
+
+
+func _process(_delta):
+	if Input.is_action_just_pressed("inventory"):
+		if $CanvasLayer.visible == false:
+			$CanvasLayer.visible = true
+		else:
+			$CanvasLayer.visible = false
+			$CanvasLayer/ColorRect/RichTextLabel.text = 'Select an item'
+
+func _on_Button_pressed():
+	if itemlist.get_selected_items().size() >= 1:
+		itemlist.remove_item(itemlist.get_selected_items()[0])
```

Comment:
itemlist.get_selected_items()[0]

is a bit awkward, yeah it works...but for readability can we do something like

var selectedItems= itemlist.get_selected_items()
itemlist.remove_item(selectedItems[0])